### ISSUE TITLE
Train on custom data

### ISSUE THREAD

**Issue by osamaalschame** [ultralytics/ultralytics#18019]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, 

I have trained my custom data using the YOLOv8 for segmentation task, I got 77% mAp. But now, using the update version, I try to train using same dataset with same model I used before, the accuracy goes in reversing starts with 77% and started to decrease in every epoch. for 100 epochs, what might be the problem?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @osamaalschame, thank you for bringing this up and for your interest in Ultralytics 🚀! We appreciate you sharing your challenge with the latest version of YOLOv8.

For custom training queries like this, it’s important to provide as much detail as possible to help diagnose the issue. Could you please share the following?  
1. Your training command or configuration details.  
2. A snippet or description of your dataset structure.  
3. Any relevant training logs or outputs, especially for the epochs where accuracy starts to decrease.  
4. Whether you’ve made any changes to default model parameters or augmentations.  

If you believe this is a 🐛 bug, kindly submit a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and reproduce the issue.

Here are a few steps you can try in the meantime to verify if the issue persists:  

## Upgrade  
Ensure you’re using the latest `ultralytics` package along with updated dependencies to rule out discrepancies with older versions:  
```bash
pip install -U ultralytics
```

## Troubleshooting Tips  
- Revisit your dataset preparation. Ensure your annotations remain consistent with the YOLO format.  
- Compare your old training configuration (before version update) with your current settings. Differences in hyperparameters, preprocessing, or augmentations can lead to varying results.  

For additional help, our documentation has all the resources you need! We recommend exploring:  
- [Training Tips for Best Results](https://docs.ultralytics.com/guides/model-training-tips/)  
- [CLI Usage for YOLOv8 Training](https://docs.ultralytics.com/usage/cli/)  

## Community Support  
Engage with our vibrant community for insights and collaborative problem-solving:  
- Join us on [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time chat.  
- Share your experiences on [Discourse](https://community.ultralytics.com).  
- Dive into community discussions and threads on our [Subreddit](https://reddit.com/r/Ultralytics).  

## Verified Environments  
You can also try training on our verified environments to ensure a standard setup:  
- **Notebooks** with free GPU:  
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>  
- **Cloud/VM Environments**: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/) or [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)  
- **Docker**: <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>  

## Status  
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>  
Our CI tests ensure YOLO works correctly across macOS, Windows, and Ubuntu. Feel free to verify your environment against the latest test runs.

This is an automated response to help streamline common queries 😊. An Ultralytics engineer will review your issue and provide further assistance shortly!

**Comment by Y-T-G** [comment]
Are you starting with your trained model?

**Comment by osamaalschame** [comment]
> Are you starting with your trained model?

both are examined, from the beginning and from trained model

**Comment by osamaalschame** [comment]
> You're getting 77 mAP when starting with the default pretrained models in the first epoch?

yes when use the trained model for first epoch, and dramatically, started to decrease for next 50~100 epochs

**Comment by osamaalschame** [comment]
> You're getting 77 mAP when starting with the default pretrained models in the first epoch?

yes when use the trained model for first epoch, and dramatically, started to decrease for next 50~100 epochs. if started from beginning, started with 50% mAp and started to decrease for next 50~100 epochs.

**Comment by osamaalschame** [comment]
> You're getting 77 mAP when starting with the default pretrained models in the first epoch?

if started from beginning, started with 50% mAp and started to decrease for next 50~100 epochs.

**Comment by Y-T-G** [comment]
Can you provide the output after running this command in terminal: `yolo checks`?

**Comment by osamaalschame** [comment]
> yolo checks

numpy               ✅ 1.24.4>=1.23.0
numpy               ✅ 1.24.4<2.0.0; sys_platform == "darwin"
matplotlib          ✅ 3.7.3>=3.3.0
opencv-python       ✅ 4.9.0.80>=4.6.0
pillow              ✅ 10.0.1>=7.1.2
pyyaml              ✅ 6.0.1>=5.3.1
requests            ✅ 2.31.0>=2.23.0
scipy               ✅ 1.10.1>=1.4.1
torch               ✅ 2.0.1>=1.8.0
torch               ✅ 2.0.1!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         ✅ 0.15.2>=0.9.0
tqdm                ✅ 4.66.1>=4.64.0
psutil              ✅ 5.9.5
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 1.2.4>=1.1.4
seaborn             ✅ 0.12.2>=0.11.0
ultralytics-thop    ✅ 2.0.10>=2.0.0

**Comment by Y-T-G** [comment]
The full output

**Comment by osamaalschame** [comment]
> The full output

Ultralytics 8.3.39 🚀 Python-3.8.10 torch-2.0.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3060)
Setup complete ✅ (16 CPUs, 125.7 GB RAM)

OS                  Linux-5.15.0-125-generic-x86_64-with-glibc2.29
Environment         Linux
Python              3.8.10
Install             pip
RAM                 125.66 GB
CPU                 11th Gen Intel Core(TM) i9-11900K 3.50GHz
CPU count           16
GPU                 NVIDIA GeForce RTX 3060
GPU count           1
CUDA                11.7

numpy               ✅ 1.24.4>=1.23.0
numpy               ✅ 1.24.4<2.0.0; sys_platform == "darwin"
matplotlib          ✅ 3.7.3>=3.3.0
opencv-python       ✅ 4.9.0.80>=4.6.0
pillow              ✅ 10.0.1>=7.1.2
pyyaml              ✅ 6.0.1>=5.3.1
requests            ✅ 2.31.0>=2.23.0
scipy               ✅ 1.10.1>=1.4.1
torch               ✅ 2.0.1>=1.8.0
torch               ✅ 2.0.1!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         ✅ 0.15.2>=0.9.0
tqdm                ✅ 4.66.1>=4.64.0
psutil              ✅ 5.9.5
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 1.2.4>=1.1.4
seaborn             ✅ 0.12.2>=0.11.0
ultralytics-thop    ✅ 2.0.10>=2.0.0

**Comment by Y-T-G** [comment]
What's the training code?

**Comment by osamaalschame** [comment]
model.train(data="data.yaml", epochs=500,imgsz=1024,resume=False,pretrained=True,
            batch=0.9,overlap_mask=True,patience=100,cache=True,optimizer='auto',plots=True,exist_ok=True,
            name='V1.0')

**Comment by glenn-jocher** [comment]
Your training argument `batch=0.9` is incorrect. Batch size must be an integer (e.g., `batch=16`). Also, ensure `data.yaml` is correctly configured for your segmentation dataset. For guidance, see [Ultralytics Train Docs](https://docs.ultralytics.com/modes/train/). Let me know if issues persist!
